src/qmapshack/gis/prj/CDetailsPrj.cpp

```diff
@@ -496,7 +496,8 @@ void CDetailsPrj::drawByGroup(QTextCursor& cursor,
 
     if(!wpts.isEmpty())
     {
-        cursor.insertHtml(tr("<h2>Waypoints</h2>"));
+        QString htmlStr = QString("<h2>%1 ").arg(wpts.count()) + tr("Waypoints</h2>");
```

Comment:
Can we make it a bit easier for translation:

`const QString& htmlStr = QString("<h2>%1 %2</h2>").arg(wpts.count()).arg(tr(Waypoints));`

Reply:
Done.